src/app/(nousSuivre)/blog/page.js

```
@@ -249,72 +257,125 @@ Grâce à la générosité de la Fondation Achraf Hakimi, nos jeunes ont été a
+                  variants={fadeIn}
+                >
+                  <div>
+                    <h1 className="text-2xl md:text-4xl font-bold text-gray-900 mb-4">
+                      {item.title}
+                    </h1>
+                    <p className="text-gray-600 mb-6 text-sm md:text-base"
+                      dangerouslySetInnerHTML={{ __html: item.shortContent }}>
+                    </p>
+                  </div>
+                  <div className="mt-auto">
+                    <a href={`/dashboard/news/${item.id}`} target="_blank" rel="noopener noreferrer">
```

Biome (1.9.4)

[error] Avoid passing both children and the dangerouslySetInnerHTML prop.

This is the source of the children prop

Setting HTML content will inadvertently override any passed children in React

(lint/security/noDangerouslySetInnerHtmlWithChildren)